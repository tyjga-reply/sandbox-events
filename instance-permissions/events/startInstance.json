{
    "version": "0",
    "id": "fbbe254c-de4b-6346-8438-b76786520b08",
    "detail-type": "AWS API Call via CloudTrail",
    "source": "aws.ec2",
    "account": "211125361846",
    "time": "2024-12-16T08:55:25Z",
    "region": "us-east-1",
    "resources": [],
    "detail": {
        "eventVersion": "1.10",
        "userIdentity": {
            "type": "AssumedRole",
            "principalId": "AROATCKANLS3NIDXYUGTV:t.kapp@reply.de",
            "arn": "arn:aws:sts::211125361846:assumed-role/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4/t.kapp@reply.de",
            "accountId": "211125361846",
            "accessKeyId": "ASIATCKANLS3MGBKRCZS",
            "sessionContext": {
                "sessionIssuer": {
                    "type": "Role",
                    "principalId": "AROATCKANLS3NIDXYUGTV",
                    "arn": "arn:aws:iam::211125361846:role/aws-reserved/sso.amazonaws.com/eu-central-1/AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4",
                    "accountId": "211125361846",
                    "userName": "AWSReservedSSO_AdministratorAccess_d519e2a9ed8243a4"
                },
                "attributes": {
                    "creationDate": "2024-12-16T08:41:38Z",
                    "mfaAuthenticated": "false"
                }
            }
        },
        "eventTime": "2024-12-16T08:55:25Z",
        "eventSource": "ec2.amazonaws.com",
        "eventName": "StartInstances",
        "awsRegion": "us-east-1",
        "sourceIPAddress": "91.102.9.132",
        "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/18.1.1 Safari/605.1.15",
        "requestParameters": {
            "instancesSet": {
                "items": [
                    {
                        "instanceId": "i-06dd978c1574e2359"
                    }
                ]
            }
        },
        "responseElements": {
            "requestId": "c94d83a7-4f46-45d7-bc2b-f877f13c1669",
            "instancesSet": {
                "items": [
                    {
                        "instanceId": "i-06dd978c1574e2359",
                        "currentState": {
                            "code": 0,
                            "name": "pending"
                        },
                        "previousState": {
                            "code": 80,
                            "name": "stopped"
                        }
                    }
                ]
            }
        },
        "requestID": "c94d83a7-4f46-45d7-bc2b-f877f13c1669",
        "eventID": "302b3213-5007-489e-9aa8-6b0ee2c0304c",
        "readOnly": false,
        "eventType": "AwsApiCall",
        "managementEvent": true,
        "recipientAccountId": "211125361846",
        "eventCategory": "Management",
        "tlsDetails": {
            "tlsVersion": "TLSv1.3",
            "cipherSuite": "TLS_AES_128_GCM_SHA256",
            "clientProvidedHostHeader": "ec2.us-east-1.amazonaws.com"
        },
        "sessionCredentialFromConsole": "true"
    }
}