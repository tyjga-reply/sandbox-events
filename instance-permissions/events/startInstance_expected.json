{"statusCode": 200, "body": "{\"message\": \"Resource ID extracted successfully!\", \"eventName\": \"StartInstances\", \"resourceId\": \"i-06dd978c1574e2359\", \"valid\": true}"}